### ISSUE TITLE
Test failure: tracing/eventpipe/reverseouter/reverseouter/reverseouter.sh

### ISSUE THREAD

**Issue by BruceForstall** [dotnet/runtime#38843]
Timeout

`CoreCLR OSX x64 Checked zapdisable @ OSX.1013.Amd64.Open`

With:
```
export COMPlus_TieredCompilation=0
export COMPlus_ReadyToRun=0
export COMPlus_ZapDisable=1
```

https://dev.azure.com/dnceng/public/_build/results?buildId=716978&view=ms.vss-test-web.build-test-results-tab&runId=22219026&resultId=109347&paneView=debug

@josalem

**Comment by josalem** [comment]
Thanks, this is on my radar.  Work is tracked here: #38156

**Comment by josalem** [comment]
Fixed by #39129
